pkg/pyinst/cryptography/osrandom_engine.c

```diff
@@ -0,0 +1,167 @@
+static const char *Cryptography_osrandom_engine_id = "osrandom";
```

Comment:
will delete this folder, it's from a previous pyinstaller release in which the data wasn't copied.